index.html

```diff
@@ -424,8 +424,8 @@ <h1>Terminology</h1>
             <dt>
                 <dfn>Execution Environment</dfn>
             </dt>
-            <dd>A sandbox within the WoT Runtime that isolates scripts
-                running on the same Servient.</dd>
+            <dd>A sandbox within the WoT Runtime that isolates application logic
```

Comment:
I would avoid saying "sandbox" and I would define the term "Execution Context" rather than "Execution Environment". Only one agent is running in one context, there could be multiple contexts but one "running execution context". Usually an execution context has associated an execution environment (global or local), eventual global objects, intrinsics, call stack, callback queue etc so it's a larger concept.

Reply:
Execution Context is too scripting-specific. I think for the general meaning we need here, Execution Environment is fine (compare, for instance, with Trusted Execution Environment)

Reply:
Execution Context and also Execution Environment are _supposed to be_ code specific, of course.
They are not scripting-specific, as you have them with native code as well.

Environment is a small part of an execution context.